### PR TITLE
LPD-33503 Order Details - Questions & Answers(order notes)

### PR DESCRIPTION
**Pull request by monaco-fabio**
https://liferay.atlassian.net/browse/LPD-33503

 Resend of #5399

### CODE REVIEW

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:relevant

**Comment by monaco-fabio** [comment]
ci:test:sf

**Comment by monaco-fabio** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/js/info_box/modal/InfoBoxModalNotesInput.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+						) : null;
+					})}
+				</div>
+			</InfiniteScrollerComponent>
+
+			<ClayInput.Group className="commerce-panel">
+				<ClayInput.GroupItem>
+					<ClayInput
+						aria-required={true}
+						className="field form-control lfr-textarea"
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/js/info_box/modal/InfoBoxModalNotesInput.js:128]: @gianmarcobrunialti please remember to add the missing css class to this text area

**Comment by gianmarcobrunialti** [comment]
Super cool, just a couple of fixes! @monaco-fabio @andrea-ale-sbarra

**Comment by andrea-ale-sbarra** [comment]
sent [here](https://github.com/liferay-commerce/liferay-portal/pull/5509)

**Pull request closed without merging**